### ISSUE TITLE
Question: May I be contribute to this repo (i think this project no more maintained)?

### ISSUE THREAD

**Issue by themisir** [waqaskhan540/IdentityServerExternalAuth#8]
I have improved this library, added support for AspNet Identity and more improvements.

**Comment by waqaskhan540** [comment]
@TheMisir thanks for showing your interest, I am not able to maintain the repo due to lack of time. It will be great if you could make some positive contributions towards the library. 
Let me know if you need any help in this regard.

**Comment by themisir** [comment]
Does this package works with .net core 2+?

**Comment by waqaskhan540** [comment]
I don't think it will be compatible with .net core 2, as I built this using .net core 1.1.

I have took the code to a nuget package , which I guess is compatible with .net core 2
here is the link https://github.com/waqaskhan540/identityserver-token-exchange
